Add CREDEBL UI User Guide documentation

konaspandana017 opened the pull request:
Added a comprehensive user guide covering the CREDEBL UI modules including Users, Ledger, Organizations, Connections, Issuance, and Verification.  Includes navigation steps, best practices, troubleshooting tips, and relevant resource links.

konaspandana017 committed:
Add CREDEBL UI User Guide documentation

Added a comprehensive user guide covering the CREDEBL UI modules including Users, Ledger, Organizations, Connections, Issuance, and Verification. 
Includes navigation steps, best practices, troubleshooting tips, and relevant resource links.


Signed-off-by: SpandanaKona <2400032836@kluniversity.in>

konaspandana017 commented:
Hi ,

All checks have passed, there are no conflicts, and the changes are ready to merge. This PR adds a comprehensive CREDEBL UI User Guide covering Users, Ledger, Organizations, Connections, Issuance, and Verification.

Could you please review and merge it at your earliest convenience?

Thank you!

konaspandana017 commented:
Hi team,

All checks have passed and the PR has no conflicts.  
The documentation is complete and ready to be merged.

Could you please review and merge it when you have time?  
Thank you!

GHkrishna commented:
Hey @konaspandana017 

Thanks for the PR

We are just checking some configurations with the tool we are using for maintaining the docs. We can merge the PR as soon as we have that sorted.

GHkrishna commented on CREDEBL-UI-User-Guide.md:
I was thinking if we could maybe add this as a page inside the  [guides](https://docs.credebl.id/docs/resources/guides) section. Just rename the section as UI User guide. Or maybe add a subsection under it. Even though I'm not really sure what other guides we might have apart from the UI User guide for now

GHkrishna commented on CREDEBL-UI-User-Guide.md:
```diff
@@ -0,0 +1,74 @@
+## Accessing the Platform
+1. Open your browser.
+2. Navigate to your environment:
+   - Local development: http://localhost:5000
```
```suggestion
   - Local development: http://localhost:5000/api
```

GHkrishna replied:
Its not currently auto redirected to `/api` for now. Have a look at this issue: https://github.com/credebl/platform/issues/1527

GHkrishna commented on CREDEBL-UI-User-Guide.md:
```diff
@@ -0,0 +1,74 @@
+2. Navigate to your environment:
+   - Local development: http://localhost:5000
+   - Hosted environment: as provided by your administrator
+3. Log in using your credentials or developer account for local setups.
```
> or developer account

Do you mean using the client credentials here?

GHkrishna commented on CREDEBL-UI-User-Guide.md:
```diff
@@ -0,0 +1,74 @@
+
+## Accessing the Platform
+1. Open your browser.
+2. Navigate to your environment:
```
I'm a bit unsure of why we are navigating to the backend if its the UI guide??

For UI, we can have localhost:3000

GHkrishna commented on CREDEBL-UI-User-Guide.md:
```diff
@@ -0,0 +1,74 @@
+
+### Organizations
+- **Add Organization**: Provide name, type, and assign credentials.
+- **Manage Credentials**: Assign, revoke, or view credentials for organizations.
```
> Assign, revoke, or view credentials for organizations.

Not sure about this. 

Do you mean inviting user and giving them different roles? (ORG invitation part)

GHkrishna commented on CREDEBL-UI-User-Guide.md:
```diff
@@ -0,0 +1,74 @@
+### Issuance & Verification
+- **Issue Credential**:
+  1. Select credential type.
+  2. Choose recipient (user or organization).
```
> organization

I'm not sure if we have this ability yet.

Can you please elaborate what exactly you mean here

GHkrishna commented on CREDEBL-UI-User-Guide.md:
```diff
@@ -0,0 +1,74 @@
+  2. Choose recipient (user or organization).
+  3. Fill required fields.
+  4. Click Issue Credential.
+- **Verify Credential**:
```
Actually here a more suited word would be sending a verification request similar to that of issuing credential

GHkrishna commented on CREDEBL-UI-User-Guide.md:
```diff
@@ -0,0 +1,74 @@
+- **Verify Credential**:
+  1. Enter credential ID.
+  2. System checks ledger.
+  3. Displays verification result.
```
It makes sense her ig to have some hints of using a wallet to store credentials. Something like https://play.google.com/store/apps/details?id=id.credebl.adeya and https://apps.apple.com/in/app/sovio-wallet-adeya-wallet/id6463845498

GHkrishna commented on CREDEBL-UI-User-Guide.md:
```diff
@@ -0,0 +1,74 @@
+- CREDEBL GitHub Repository: https://github.com/credebl/platform
+- Backend Setup & Environment Guide: README.md
+- Docker Documentation: https://docs.docker.com/
+- NATS Documentation: https://docs.nats.io/
```
Why this links, maybe having link to the [local-deployment](https://docs.credebl.id/docs/getting-started/local-deployment) section would be more appropriate

konaspandana017 committed:
Revise CREDEBL UI User Guide for clarity and updates

Updated the CREDEBL UI User Guide to include versioning, improved instructions, and additional details for various sections.

Signed-off-by: SpandanaKona <2400032836@kluniversity.in>

ajile-in commented:
Closed due to no further updates.
